pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -80,6 +80,12 @@ func (a *ofFlowAction) CT(commit bool, tableID uint8, zone int, zoneSrcField *Re
 	return ct
 }
 
+// FieldPair is a struct to store source reg mark field and destination CT mark field for moving.
+type FieldPair struct {
```

Comment:
So it is required the source field must be register, and the value is ct_mark? The usage is limited?

Reply:
Yes.